.mailmap

```diff
@@ -1,7 +1,7 @@
 Tomer Doron <tomerd@apple.com> <tomer.doron@gmail.com>
 Tomer Doron <tomerd@apple.com>
 Max Moiseev <moiseev@apple.com>
-Johannes Weiß <johannesweiss@apple.com>
+Johannes Weiss <johannesweiss@apple.com> Johannes Weiß <johannesweiss@apple.com>
```

Comment:
This change brought to you by HM Government.

Reply:
😂